### PR TITLE
Forward port of 8.6 and 8.5.3 release notes to main

### PR DESCRIPTION
**Pull request by andsel**
## Release notes
[rn:skip]

## What does this PR do?
Update release notes to include recent changes for `8.6` and `8.5.3`

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -29,6 +31,72 @@ This section summarizes the changes in the following releases:
...
+=== Logstash 8.6.0 Release Notes
+
+[[features-8.6.0]]
+==== New features and enhancements
+
+* Extends the flow rates introduced to the Node Stats API in 8.5.0 (which included windows for `current` and `lifetime`)
+to include a Technology Preview of several additional windows such as `last_15_minutes`, `last_24_hours`, etc..
+https://github.com/elastic/logstash/pull/14571[#14571]
+* Logstash introduced instance and pipeline level flow metrics, growth_bytes and growth_events for persisted queue
+to provide a better visibility about how fast pipeline queue is growing each single seconds.
```
Comment (mashhurs) [inline docs/static/releasenotes.asciidoc:44]:
```suggestion
* Logstash introduced instance and pipeline level flow metrics, `growth_bytes` and `growth_events` for persisted queue
to provide a better visibility about how fast pipeline queue is growing.
```

**Review by mashhurs — COMMENTED** [review]
[A poller collects the flow metrics every 5sec](https://github.com/mashhurs/logstash/blob/e121650e56d7e79d9692ba34f117ce683f9e5f7c/logstash-core/lib/logstash/instrument/periodic_poller/base.rb#L26), better to remove every second statement I think.

File: docs/static/releasenotes.asciidoc (hunk 1 above)
Comment (andsel) [reply docs/static/releasenotes.asciidoc:44]: The PR is just a forward port of #14762 which is already merged in `8.6`. 
I think that if we change her we should also backport then to the `8.6`.

**Review by mashhurs — APPROVED** [review]
LGTM!

File: docs/static/releasenotes.asciidoc (hunk 1 above)
Comment (mashhurs) [reply docs/static/releasenotes.asciidoc:44]: Ah if we need to change, we have to both...

**Comment by andsel** [comment]
Note, :red_circle:  CI is described in #14831

**Pull request merged**